<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employee_Calendar</class>
 <widget class="QMainWindow" name="Employee_Calendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee_Calendar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>701</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>130</y>
      <width>151</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Off Days </string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitBtn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>530</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit Availablity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>                          Enter ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Press to Continue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>511</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>selectionChanged()</signal>
   <receiver>Employee_Calendar</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>Employee_Calendar</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submitBtn</sender>
   <signal>clicked()</signal>
   <receiver>Employee_Calendar</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>825</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Employee_Calendar</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
